<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeChooser</class>
 <widget class="QDialog" name="ShapeChooser">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Your Shapes</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="polygonCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>16</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Polygons</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="rectangleCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Rectangles</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="ellipseCheckBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>16</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ellipses</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="lineCheckBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>48</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Lines</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShapeChooser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShapeChooser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
